pytorch_toolkit/object_detection/face-detection/get_size.py

```diff
@@ -0,0 +1,30 @@
+import os
```

Comment:
Why this script is not a part of the tools?

Reply:
accidentally added. removed